radiacode_examples/webserver_logger.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
```

Comment:
I think we can choose a better name for this example. I find the CPS notifications are more important than the logging ability, so maybe choose something neutral or showing both (logs & CPS alerts)?